source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -5,15 +5,23 @@ protocol ACRTextFieldDelegate: AnyObject {
     func acrTextFieldDidSelectClear(_ textField: ACRTextField)
 }
 
+private enum textFieldMode {
+    case inputText
```

Comment:
👍

Reply:
```suggestion
private enum Mode {
    case inputText
```

This enum can be inside `ACRTextField` class.